ROADMAP.md

```diff
@@ -1,56 +1,75 @@
 # Metal3 Roadmap
 
 The Metal3 Roadmap is maintained as a GitHub project and can be found
-[here](https://github.com/orgs/metal3-io/projects/2).
+[here](https://github.com/orgs/metal3-io/projects/8).
 
 ## Description
 
-Each column in the project represents the work items for a specific release of
-either Baremetal Operator, Cluster API Provider Metal3, IP Address Manager,
+Each column in the project represents the work items for either
+Baremetal Operator, Cluster API Provider Metal3, IP Address Manager,
 Ironic image and other Metal3 components. In addition there is
-a `Feature Requests` column that contains items that have not yet been
+a `No status` column that contains items that have not yet been
 accepted and the `Backlog` column that contains items that have been accepted
-but not yet planned for a specific release.
+but work has not started on them.
 
 An issue can be planned for a specific release if someone volunteers to take
 ownership of the feature. The owner will then be assigned the issue. An owner
```

Comment:
```suggestion
ownership of the issue. The owner will then be assigned the issue. An owner
```
Not all issues on roadmap are features, if we put all new issues there as you suggested yesterday.